types/yup/index.d.ts

```diff
@@ -12,7 +12,7 @@
 //                 Desmond Koh <https://github.com/deskoh>
 //                 Maurice de Beijer <https://github.com/mauricedb>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 2.9
```

Comment:
Just curious, what necessitates the bump to 2.9?

Reply:
@andrewbranch When doing this update I had a error which was fixed by updating the TS version to 2.9. But trying to reproduce the same error now by reverting the version update gives me no error. So I will revert this change.